dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/process/ProcessService.java

```diff
@@ -575,23 +596,6 @@ private ProcessInstance generateNewProcessInstance(ProcessDefinition processDefi
                 }
             }
         }
-
```

Comment:
This piece of code should remain unchanged

Reply:
@CalvinKirs 
Sure, that piece of code was rolled back to it's origin status.
Only those which related to repeat running were changed.

Also, I have tested recover failed function of a process instance, the same problem occured as this issue.
Should I create another issue to report this situation?

Reply:
sure, one issue should correspond to one pr, welcome submit

Reply:
So far I've met a little question about code coverage failed.
I've tested my commitment in my produce environment.
But if I wanted to submit this PR, I didn't pass the sonar Analysis.
If I want to create Unit Test to cover my code, I have to specify configurations such as DB and zookeeper, I tried to copy exsisting unit test code and changed the variables, it seems not working.